guides/doc-Project_API/topics/con_ssl-authentication-overview.adoc

```diff
@@ -0,0 +1,8 @@
+[id="ssl-authentication-overview"]
+= SSL authentication overview
+
+{ProjectName} uses HTTPS, which provides a degree of encryption and identity verification when communicating with a {ProjectServer}.
+{Project} {ProjectVersion} does not support non-SSL communications.
+
+Each {ProjectServer} uses a self-signed certificate.
```

Comment:
```suggestion
By default, each {ProjectServer} uses a self-signed certificate.
```

I think it's quite common to provide custom SSL certs during installation/later on. Maybe we should reference these procedures: https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#Configuring_Server_with_a_Custom_SSL_Certificate_foreman

Reply:
Done.